kvbc/include/kvbc_key_types.hpp

```diff
@@ -27,6 +29,7 @@ static const char reconfiguration_tls_exchange_key = 0x2e;
 
 static const char reconfiguration_restart_key = 0x30;
 static const char reconfiguration_ts_key = 0x31;
+static const std::string genesis_block_key(1, 0x32);
```

Comment:
All reconfiguration keys are now being written under the reconfiguration category, so you can reuse keys under a different category if you want.